demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -678,7 +677,6 @@ export class DemoMeetingApp
   initEventListeners(): void {
     if (!this.defaultBrowserBehaviour.hasChromiumWebRTC()) {
       (document.getElementById('simulcast') as HTMLInputElement).disabled = true;
-      (document.getElementById('planB') as HTMLInputElement).disabled = true;
```

Comment:
Can you remove the element in the html file also?

Reply:
I thought I already removed it?